### PR TITLE
Add sockaddr_storage

### PR DESCRIPTION
**Pull request by fjtrujy**

### CODE REVIEW

**Review by diamant3 — APPROVED** [review]

**Comment by sajattack** [comment]
Wanna maybe give us some clue what this is for?

File: src/libcglue/sys/socket.h [hunk 1]
```diff
@@ -169,6 +169,22 @@ struct sockaddr {
...
+/*
+ * RFC 2553: protocol-independent placeholder for socket addresses
+ */
+#define _SS_MAXSIZE	128
+#define _SS_ALIGNSIZE	(sizeof(__int64_t))
+#define _SS_PAD1SIZE	(_SS_ALIGNSIZE - 2)
+#define _SS_PAD2SIZE	(_SS_MAXSIZE - 2 - _SS_PAD1SIZE - _SS_ALIGNSIZE)
+
+struct sockaddr_storage {
+	__uint8_t	ss_len;		/* address length */
```
Comment (rofl0r) [inline src/libcglue/sys/socket.h:181]: struct sockaddr itself uses plain `uint8_t` so i suppose it should be ok to not use `__`-prefixed typenames

**Comment by fjtrujy** [comment]
> Wanna maybe give us some clue what this is for?

I was suffering some errors compiling `WolfSSL` for use with `cURL`.

https://github.com/wolfSSL/wolfssl/blob/master/wolfssl/wolfio.h#L418

File: src/libcglue/sys/socket.h (hunk 1 above)
Comment (fjtrujy) [reply src/libcglue/sys/socket.h]: Done!

**Pull request merged**